0miner

```diff
@@ -41,481 +54,345 @@ then
   MINER_PWD="x"
 fi
 
-if [ $COIN == "ONION" ]
-then
-  HCD='/home/m1/TPccminer/ccminer'
-  ADDR="$ONION_ADDRESS"
-  screen -dmSL miner $HCD -a x13 -o stratum+tcp://$ONION_POOL:$ONION_PORT -u $ADDR -p c=ONION
-fi
+## Unify COIN Start
+WALLET_ADDRESS_FORMAT="."
+UPOOL="_POOL"
+UPORT="_PORT"
+UEXT="_EXTENSION_ARGUMENTS"
+UADDR="_ADDRESS"
+UWORK="_WORKER"
+LAUNCH="screen -dmSL miner"
+xpool=$COIN$UPOOL
+xport=$COIN$UPORT
+xadd=$COIN$UADDR
+xwrk=$COIN$UWORK
+xext=$COIN$UEXT
+CLN=:
+## Unify COIN End
 
-if [ $COIN == "SMART" ]
-then
-  HCD='/home/m1/ASccminer/ccminer'
-  ADDR="$SMART_ADDRESS.$SMART_WORKER"
-  screen -dmSL miner $HCD -a keccak -o stratum+tcp://$SMART_POOL:$SMART_PORT -u $ADDR -p $MINER_PWD -i $SMART_INTENSITY
+## NICEHASH START
+if [ $COIN == "NICE_NEOSCRYPT" -o $COIN == "NICE_LYRA2REV2" -o $COIN == "NICE_X11GOST" -o $COIN == "NICE_SKUNKHASH" -o $COIN == "NICE_CRYPTONIGHT"  -o $COIN == "NICE_EQUIHASH" -o $COIN == "NICE_ETHASH" ]
+  then
+    NICE_ADDRESS=$BTC_ADDRESS
+	NICE_WORKER=$WORKERNAME
+	NICE_NEOSCRYPT_ADDRESS=$BTC_ADDRESS
+	NICE_LYRA2REV2_ADDRESS=$BTC_ADDRESS
+	NICE_X11GOST_ADDRESS=$BTC_ADDRESS
+	NICE_SKUNKHASH_ADDRESS=$BTC_ADDRESS
+	NICE_CRYPTONIGHT_ADDRESS=$BTC_ADDRESS
+	NICE_EQUIHASH_ADDRESS=$BTC_ADDRESS
+	NICE_ETHASH_ADDRESS=$BTC_ADDRESS
 fi
+## NICEHASH END
 
-if [ $COIN == "BTG" ]
-then
-  BTGADDR="$BTG_ADDRESS.$BTG_WORKER"
-  if [ $EWBF_VERSION == "3_3" ]
-  then
-    HCD='/home/m1/zec/v3_3/miner'
-  fi
-  if [ $EWBF_VERSION == "3_4" ]
-  then
-    HCD='/home/m1/zec/v3_4/miner'
-  fi
-  if [ $EQUIHASH_MINER == "ZM" ]
-  then
-    HCD='/home/m1/zec/zm/latest/zm_miner'
-    screen -dmSL miner $HCD --server $BTG_POOL --user $BTGADDR --pass $MINER_PWD --port $BTG_PORT  --time $ZM_OPTS ;
-  fi
-  if [ $EQUIHASH_MINER == "EWBF" ]
+## IDENTIFY ALGORITHM START
```

Comment:
Do we need this algorithm identify?
We already have it in 1bash

Reply:
Got it, I have a nice idea to rework the algo-specific identification in 1bash, moving further towards ultimate OC settings solution. I can change that in my OC policy branch but I need this @sizzlephizzle implementation to be merged.